python-package/lightgbm/engine.py

```diff
@@ -131,12 +131,16 @@ def train(params, train_data, num_boost_round=100,
         predictor = init_model
     else:
         predictor = None
+    init_iteration = predictor.init_iteration if predictor else 0
```

Comment:
OK, i see. 
I think maybe we should start from init_iteration in here:
https://github.com/Microsoft/LightGBM/blob/master/python-package/lightgbm/engine.py#L212